ERCS/erc-xxxx.md

```diff
@@ -0,0 +1,262 @@
+---
+title: Routing Interface Standard
```

Comment:
```suggestion
title: Routing Interface
```
Standard is superfluous in an ERC.
Perhaps use something more descriptive such as "swap order routing"?

Reply:
Thank you!